tests/Api/Shop/OrdersTest.php

```diff
@@ -220,4 +221,54 @@ public function it_allows_to_patch_orders_payment_method(): void
 
         $this->assertResponse($response, 'shop/updated_payment_method_on_order_response', Response::HTTP_OK);
     }
+
+    /** @test */
+    public function it_allows_to_patch_orders_address(): void
+    {
+        $fixtures = $this->loadFixturesFromFiles(['authentication/customer.yaml', 'channel.yaml', 'cart.yaml', 'country.yaml', 'shipping_method.yaml', 'payment_method.yaml']);
+
+        $tokenValue = 'nAWw2jewpA';
+
+        /** @var MessageBusInterface $commandBus */
+        $commandBus = $this->get('sylius.command_bus');
+
+        $pickupCartCommand = new PickupCart($tokenValue);
+        $pickupCartCommand->setChannelCode('WEB');
+        $commandBus->dispatch($pickupCartCommand);
+        $addItemToCartCommand = new AddItemToCart('MUG_BLUE', 3);
+        $addItemToCartCommand->setOrderTokenValue($tokenValue);
+        $commandBus->dispatch($addItemToCartCommand);
+
+        /** @var CountryInterface $country */
+        $country = $fixtures['country_US'];
+
+        $billingAddress = [
+            'firstName'=> 'Jane',
+            'lastName'=> 'Doe',
+            'phoneNumber'=> '666111333',
+            'company'=> 'Potato Corp.',
+            'countryCode'=> $country->getCode(),
+            'street'=> 'Top secret',
+            'city'=> 'Nebraska',
+            'postcode'=> '12343'
+        ];
+
+        $this->client->request(
+            'PUT',
+            '/api/v2/shop/orders/nAWw2jewpA',
+            [],
+            [],
+            [
+                'CONTENT_TYPE' => 'application/ld+json',
+                'HTTP_ACCEPT' => 'application/ld+json',
+            ],
+            json_encode([
+                'email' => 'oliver@doe.com',
+                'billingAddress' => $billingAddress,
+            ])
```

Comment:
IMO we can take advantage of [named parameters](https://php.watch/versions/8.0/named-parameters) to get rid of empty arguments
```suggestion
            method: 'PUT',
            uri: '/api/v2/shop/orders/nAWw2jewpA',
            server: [
                'CONTENT_TYPE' => 'application/ld+json',
                'HTTP_ACCEPT' => 'application/ld+json',
            ],
            content: json_encode([
                'email' => 'oliver@doe.com',
                'billingAddress' => $billingAddress,
            ])
```